source/blog/2018-11-16-the-ember-times-issue-73.md

```diff
@@ -43,8 +44,13 @@ Have you heard about [Hooks in React](https://reactjs.org/docs/hooks-intro.html)
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [eslint-plugin-ember v6.0.0 is out! 🚓](https://twitter.com/TobiasBieniek/status/1062700572757946368)
+
+[@Turbo87](https://github.com/Turbo87) announced the release of [v6.0.0](https://github.com/ember-cli/eslint-plugin-ember/releases/tag/v6.0.0) of [eslint-plugin-ember] which is an [ESLint](https://eslint.org/) plugin that provides set of rules for Ember Applications based on commonly known good practices.
```

Comment:
Maybe [eslint-plugin-ember] should be surrounded by ** or `?

Reply:
Or maybe this is supposed to actually link to the addon github?

Reply:
Whoops! Missed the linking!